packages/browser/src/integrations/trycatch.ts

```diff
@@ -148,14 +148,12 @@ function _wrapRAF(original: any): (callback: () => void) => any {
 function _wrapXHR(originalSend: () => void): () => void {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   return function (this: XMLHttpRequest, ...args: any[]): void {
-    // eslint-disable-next-line @typescript-eslint/no-this-alias
-    const xhr = this;
     const xmlHttpRequestProps: XMLHttpRequestProp[] = ['onload', 'onerror', 'onprogress', 'onreadystatechange'];
 
     xmlHttpRequestProps.forEach(prop => {
-      if (prop in xhr && typeof xhr[prop] === 'function') {
+      if (prop in this && typeof this[prop] === 'function') {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        fill(xhr, prop, function (original: WrappedFunction): () => any {
+        fill(this, prop, (original: WrappedFunction): (() => any) => {
```

Comment:
I am not 100% sure if it is safe to replace these with anonymous functions - we should just double check, I remember seeing some comments somewhere that in some cases this may be tricky.